packages/client/hmi-client/src/services/project.ts

```diff
@@ -93,6 +93,7 @@ async function getAll(): Promise<Project[] | null> {
  * @return ProjectAssets|null - the appropriate project, or null if none returned by API
  */
 async function getAssets(projectId: string, types?: string[]): Promise<ProjectAssets | null> {
+	if (projectId === '') return null;
```

Comment:
This seems unrelated, what are you trying to do?

Reply:
will remove, I was debugging the 404 error when projects weren't appearing